bin/make-base-vm

```diff
@@ -153,6 +159,32 @@ fi
 # Remove cron to work around vmbuilder issue when umounting /dev on target
 removepkg=cron
 
+if [ $DOCKER = "1" ]; then
+
+  addpkg=`echo $addpkg | tr ',' ' '`
+
+  mkdir -p docker
+  cd docker
+
+  # Generate the dockerfile
+  cat << EOF > $OUT.Dockerfile
+FROM $DISTRO:$SUITE
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get --no-install-recommends -y install $addpkg
+
+RUN useradd -ms /bin/bash -U ubuntu
+USER ubuntu:ubuntu
+WORKDIR /home/ubuntu
```

Comment:
Is that right, or should it be `$DISTRO` on these three lines instead of `ubuntu`?